### PR TITLE
change DashboardPageContainer.tsx, redux-store.tsx

### PR DESCRIPTION
**Pull request by LuydmilaKolesnikova**

### CODE REVIEW

File: src/components/DashboardPage/DashboardPageContainer.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { connect } from "react-redux";
 import DashboardPage from "./DashboardPage";
 import { getAuthUserDashboard } from "../../redux/dashboard-reducer";
+import { State } from "../../redux/redux-store";
 
 class DashboardPageContainer extends React.Component<any> {
```
Comment (quanterion) [inline src/components/DashboardPage/DashboardPageContainer.tsx]: get rid of any

File: src/redux/redux-store.tsx [hunk 2]
```diff
@@ -14,4 +14,10 @@ let store = legacy_createStore(reducers, applyMiddleware(thunk));
 
 (window as any).store = store;
 
+export interface State {
+  auth: {};
```
Comment (quanterion) [inline src/redux/redux-store.tsx]: why empty objects: auth, header, dashboard?

File: src/components/DashboardPage/DashboardPageContainer.tsx (hunk 1 above)
Comment (LuydmilaKolesnikova) [reply src/components/DashboardPage/DashboardPageContainer.tsx]: I only can replace any on unknown. Is it right?

Comment (quanterion) [reply src/components/DashboardPage/DashboardPageContainer.tsx]: if no properties required by component then omit generic type altogether `class DashboardPageContainer extends React.Component {`
if component has props use interface or just define it inline  `class DashboardPageContainer extends React.Component<{a: number}> {`

**Pull request merged**